R/build-reference-index.R

```diff
@@ -104,6 +104,9 @@ check_missing_topics <- function(rows, pkg) {
 
   missing <- !in_index & !pkg$topics$internal
   if (any(missing)) {
+    if (tolower(Sys.getenv("CI", "false")) == "true") {
```

Comment:
Can you please copy in `testthat:::on_ci()` and use that?

Reply:
Done in 024ac28